tests/test_config.py

```diff
@@ -224,6 +224,28 @@ def bob_config(transient_to, initial_config):
     assert "quit" not in bob.text()
 
 
+@pytest.mark.skipif(
+    os.environ["IRC_SERVER"] == "hircd",
+    reason="Hircd currently doesn't support the WHOIS command. IrcCore._nickmask will therefore always be None in the test.",
```

Comment:
If it's always `None`, you might as well assert it in the test. For example:
```python3
if os.environ["IRC_SERVER"] == "hircd":
    assert server_view.core.get_nickmask() is None
else:
    assert server_view.core.get_nickmask() == Foo!AliceUsr@127.0.0.1"
```

This ensures that `get_nickmask()` works even when the server is somewhat dumb.